WordPressShared/Core/Analytics/WPAnalytics.h

```diff
@@ -134,7 +134,6 @@ typedef NS_ENUM(NSUInteger, WPAnalyticsStat) {
     WPAnalyticsStatEnhancedSiteCreationSuccessLoading,
     WPAnalyticsStatEnhancedSiteCreationSuccessPreviewViewed,
     WPAnalyticsStatEnhancedSiteCreationSuccessPreviewLoaded,
-    WPAnalyticsStatEnhancedSiteCreationCompleted,
```

Comment:
Would it make sense to match what @malinajirka is doing with the Android patch and add a `WPAnalyticsStatEnhancedSiteCreationPreviewOKButtonTapped` event?  Not sure if the flows are similar enough to warrant it?

Reply:
Good point, I've added the stat, and updated the WPiOS branch that points to this one.

Please keep in mind the WPiOS branch still doesn't track this, but it's been updated to build - which should be enough for now, and for the purpose of just testing these changes.